tests/php/BlockExtendTestCase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace XWP\BlockExtendTest;
+
+use WP_Mock\Tools\TestCase;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+
+/**
+ * Our basic test case.
+ */
+class BlockExtendTestCase extends TestCase {
```

Comment:
This generic test case allows us to extend the features without having to update each individual test in the future.

Reply:
Nice!